tools/pinning/oldest/pyproject.toml

```diff
@@ -60,7 +60,7 @@ cffi = "1.9.1"
 chardet = "2.2.1"
 cloudflare = "1.5.1"
 configobj = "5.0.6"
-cryptography = "2.1.4"
+cryptography = "2.5.0"
```

Comment:
I think we can just unpin this, since cryptography isn't actually packaged for EPEL 8. If it were, we should hold it back to that version, but since it's not, we don't have to. Not sure where our EPEL 8 package is actually getting its cryptography install from.

Reply:
Alright, I've changed this one to 3.2.1, and then used repin.sh to regenerate the entirety of `oldest_contraints.txt`. That introduced a bunch of changes that I'm not really able to evaluate but are *probably* all fine and necessary?

I've left it set to ">=2.5.0" in the setup.py files, because I think that's still an accurate description of the true dependency restriction for those packages.

Reply:
Perfect, that all sounds correct. Launched the full test suite to double check that all the changes will be fine; we have it set up like this to mostly keep things pinned but upgrade every semi-regularly. Gave them a skim but if tests pass then all the changes are good.